Analysis/Tasks/PWGUD/upcForward.cxx

```diff
@@ -35,10 +35,12 @@ struct UPCForward {
   //defining histograms
   OutputObj<TH1D> hMass{TH1D("hMass", ";#it{m_{#mu#mu}}, GeV/c^{2};", 500, 0, 10)};
   OutputObj<TH1D> hPt{TH1D("hPt", ";#it{p_{t}}, GeV/c;", 500, 0., 5.)};
-  OutputObj<TH1D> hPx{TH1D("hPx", ";#it{p_{x}}, GeV/c;", 500, 0., 5.)};
-  OutputObj<TH1D> hPy{TH1D("hPy", ";#it{p_{y}}, GeV/c;", 500, 0., 5.)};
-  OutputObj<TH1D> hPz{TH1D("hPz", ";#it{p_{z}}, GeV/c;", 500, 0., 5.)};
+  OutputObj<TH1D> hPtsingle_muons{TH1D("hPtsingle_muons", ";#it{p_{t}}, GeV/c;", 500, 0., 5.)};
+  OutputObj<TH1D> hPx{TH1D("hPx", ";#it{p_{x}}, GeV/c;", 500, -5., 5.)};
+  OutputObj<TH1D> hPy{TH1D("hPy", ";#it{p_{y}}, GeV/c;", 500, -5., 5.)};
+  OutputObj<TH1D> hPz{TH1D("hPz", ";#it{p_{z}}, GeV/c;", 500, -5., 5.)};
   OutputObj<TH1D> hRap{TH1D("hRap", ";#it{y},..;", 500, -10., 10.)};
+  OutputObj<TH1D> hEta{TH1D("hEta", ";#it{y},..;", 500, -10., 10.)};
   OutputObj<TH1D> hCharge{TH1D("hCharge", ";#it{charge},..;", 500, -10., 10.)};
   OutputObj<TH1D> hSelectionCounter{TH1D("hSelectionCounter", ";#it{Selection},..;", 30, 0., 30.)};
   OutputObj<TH1D> hPhi{TH1D("hPhi", ";#it{#Phi},;", 500, -6., 6.)};
```

Comment:
Could you use the histogram registry? See e.g. Maja's talk yesterday at the S&C meeting for some information.

Reply:
ok I will use  the histogram registry

Reply:
I updated my code as suggested.

Reply:
I forgot to make further changes using the histogram registry. I will update my code